js/index.js

```diff
@@ -59,4 +59,25 @@ aTag[4].setAttribute('a', 'nav-item-5');
 aTag[4].innerText = 'About';
 
 aTag[5].setAttribute('a','nav-item-6');
-aTag[5].innerText ='Contact';
\ No newline at end of file
+aTag[5].innerText ='Contact';
+
+//cta section
+const newH1Tag = document.querySelector('h1');
+newH1Tag.textContent = `DOM Is Awesome`;
```

Comment:
instead of typing out the literal string `DOM Is Awesome` you could access the `SiteContent` object that contains this value. This is something you will do a lot in React using data that you pull from an API, or stored in state to create the UI of the web app.